packages/cache/src/internal/cacheUtils.ts

```diff
@@ -116,9 +120,12 @@ export function getCacheFileName(compressionMethod: CompressionMethod): string {
     : CacheFilename.Zstd
 }
 
-export async function isGnuTarInstalled(): Promise<boolean> {
+export async function getGnuTarPathOnWindows(): Promise<string> {
```

Comment:
This returns any GNU tar present in path or returns empty string is equivalent to false